source/SIL.AppBuilder.Portal.Frontend/src/ui/components/header/notifications/data.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+import { compose } from 'recompose';
+import { withData as withOrbit, WithDataProps } from 'react-orbitjs';
+import { TYPE_NAME as NOTIFICATION, NotificationAttributes } from '@data/models/notification';
+import { query, defaultSourceOptions } from '@data';
+import { withStubbedDevData } from '@data/with-stubbed-dev-data';
+import { uuid } from '@orbit/utils';
+
+// const mapNetworkToProps = (passedProps) => {
```

Comment:
comments should be removed

Reply:
or add a note to use this when we have an api endpoint to pull notifications from

Reply:
I will add the comment because we probably going to need it

Reply:
coolio